es6-shim.js

```diff
@@ -1264,10 +1264,7 @@
   // Chrome defines keys/values/entries on Array, but doesn't give us
   // any way to identify its iterator.  So add our own shimmed field.
   if (Object.getPrototypeOf) {
-    var ChromeArrayIterator = Object.getPrototypeOf([].values());
-    if (ChromeArrayIterator) { // in WSH, this is `undefined`
-      addIterator(ChromeArrayIterator);
-    }
```

Comment:
is this no longer true? iow, in WSH, `Object.getPrototypeOf([].values());` is not actually `undefined`?

Reply:
Only if you apply this change.